pkg/libvirttools/virtualization.c

```diff
@@ -20,16 +20,64 @@ limitations under the License.
 #include "virtualization.h"
```

Comment:
Is there a good reason why we wnat to have these c wrappers? For now they don't give any additional value, i mean we can do it all thru cgo in the same manner.